### ISSUE TITLE
What about 'scratch' and 'tune' models?

### ISSUE THREAD

**Issue by taimoor6864** [richzhang/PerceptualSimilarity#42]
Hey, I hope you are well. 
   In the model options for your loss, we can either go for 'net'(vanilla pre-trained CNN) or 'net-lin' (which I assume is the one with the learned linear layer). I was interested in the 'tune' and 'scratch' models of the CNNs for research purposes, are they available, how can I obtain them? Thank you for your time.

**Comment by richzhang** [comment]
Thanks for the question! They're not available at this time, but training scripts are shown in `scripts` folder.
